admin_manual/installation/system_requirements.rst

```diff
@@ -11,30 +11,30 @@ For best performance, stability and functionality we have documented some recomm
-|                  | - Debian 12 (Bookworm)                                                |
-|                  | - SUSE Linux Enterprise Server 15                                     |
-|                  | - openSUSE Leap 15.5                                                  |
-|                  | - CentOS Stream                                                       |
-+------------------+-----------------------------------------------------------------------+
-| Database         | - **MySQL 8.0+** or MariaDB 10.3/10.5/**10.6** (recommended)/10.11    |
-|                  | - Oracle Database 11g (*only as part of an enterprise subscription*)  |
-|                  | - PostgreSQL 12/13/14/15/16                                           |
-|                  | - SQLite (*only recommended for testing and minimal-instances*)       |
-+------------------+-----------------------------------------------------------------------+
-| Webserver        | - **Apache 2.4 with** ``mod_php`` **or** ``php-fpm`` (recommended)    |
-|                  | - nginx with ``php-fpm``                                              |
-+------------------+-----------------------------------------------------------------------+
-| PHP Runtime      | - 8.1 (*deprecated*)                                                  |
-|                  | - 8.2                                                                 |
-|                  | - **8.3** (*recommended*)                                             |
-+------------------+-----------------------------------------------------------------------+
++------------------+------------------------------------------------------------------------------------+
+| Platform         | Options                                                                            |
++==================+====================================================================================+
+| Operating System | - **Ubuntu 22.04 LTS** (recommended)                                               |
+| (64-bit)         | - Ubuntu 20.04 LTS                                                                 |
+|                  | - **Red Hat Enterprise Linux 9** (recommended)                                     |
+|                  | - Red Hat Enterprise Linux 8                                                       |
+|                  | - Debian 12 (Bookworm)                                                             |
+|                  | - SUSE Linux Enterprise Server 15                                                  |
+|                  | - openSUSE Leap 15.5                                                               |
+|                  | - CentOS Stream                                                                    |
++------------------+------------------------------------------------------------------------------------+
+| Database         | - **MySQL** **8.0**/8.2/8.3 or MariaDB 10.3/10.5/**10.6**/10.11 (recommended)      |
```

Comment:
I think we should only list the LTS version 8.0 and 8.4 (and add 8.4 to our testing matrix).

Reply:
Agreed.

Reply:
https://schilljs.com

Reply:
The ` (recommended)` was inline, to clarify that we recommend the 10.6 instead of any mysql/mariadb
But it seems that got lost already for mysql on the last bump. So let's hope the bolding is enough.